packages/atlas/src/components/_crt/CrtRevenueShareWidget/CrtRevenueShareWidget.tsx

```diff
@@ -0,0 +1,160 @@
+      customTopRightNode={
+        <TextButton iconPlacement="right" onClick={onTabSwitch} icon={<SvgActionChevronR />}>
+          Show revenue shares
+        </TextButton>
+      }
+      customNode={
+        <FlexBox flow="column" justifyContent="space-between" height="100%" width="100%" gap={4}>
+          <FlexBox width="100%" equalChildren justifyContent="space-between">
+            <FlexBox flow="column">
+              <Text variant="h100" as="h1" color="colorTextMuted">
+                {status === 'inactive'
+                  ? 'CURRENT STATE'
+                  : status === 'past'
+                  ? 'REVENUE SHARE ENDED ON'
+                  : 'REVENUE SHARE ENDS IN'}
+              </Text>
+              {status !== 'inactive' && activeRevenueShare?.endsAt ? (
+                status === 'past' ? (
+                  <Text variant="h400" as="h5" margin={{ bottom: 4 }}>
+                    {endingDate ? formatDateTime(endingDate).replace(',', ' at') : 'N/A'}
+                  </Text>
+                ) : (
+                  <Text variant="h400" as="h5">
+                    {endingDate ? formatDurationShort(Math.round((endingDate.getTime() - Date.now()) / 1000)) : 'N/A'}
+                  </Text>
+                )
+              ) : (
+                <Text variant="h400" as="h5" margin={{ bottom: 4 }}>
+                  No active share
+                </Text>
+              )}
+            </FlexBox>
+            <FlexBox flow="column">
+              <Text variant="h100" as="h1" color="colorTextMuted">
+                REVENUE SHARE RATIO
+              </Text>
+              <Text variant="h400" as="p">
+                Channel {token.revenueShareRatioPermill}%, Holders {100 - token.revenueShareRatioPermill}%
+              </Text>
+            </FlexBox>
+          </FlexBox>
+          <FlexBox flow="column">
+            <Text variant="h100" as="h1" color="colorTextMuted">
+              STAKED HOLDERS
+            </Text>
+            {activeRevenueShare?.stakers.length ? (
+              <StakersBox>
+                <FlexBox>
+                  {activeRevenueShare.stakers.slice(0, Math.min(activeRevenueShare.stakers.length, 5)).map((staker) => (
+                    <StakerPill key={staker.id} id={staker.account.member.id} />
+                  ))}
+                  {activeRevenueShare.stakers.length > 5 ? (
+                    <StyledPill label={activeRevenueShare.stakers.length - 5} />
+                  ) : null}
+                </FlexBox>
+              </StakersBox>
+            ) : (
+              <Text variant="h400" as="p">
+                No holders staked yet
+              </Text>
+            )}
+          </FlexBox>
+
+          {status === 'inactive' || !activeRevenueShare ? (
+            <EmptyState />
+          ) : (
+            <RevenueShareProgress revenueShare={activeRevenueShare} hasEnded={status === 'past'} />
+          )}
+
+          {status !== 'inactive' && (
```

Comment:
should this button be there for the 'past' state?

Reply:
![image](https://github.com/Joystream/atlas/assets/92513933/14729fc7-7be6-4759-a1bd-37508dcb72d3)

It's disabled on `active`